### PR TITLE
Possible fix for #25 #26

### PR DESCRIPTION
**Pull request by nathantannar4**
I can't seem to recreate the bouncing in the MessageViewController example but in my other project I saw that this helped.

TL;DR
This just enforces a set paragraph style on all attributed strings

### CODE REVIEW

**Comment by rnystrom** [comment]
Awesome! Lets give it a shot.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: MessageViewController/MessageAutocompleteController.swift [hunk 1]
```diff
@@ -46,8 +46,17 @@ public final class MessageAutocompleteController: MessageTextViewListener {
     private var typingTextAttributes: [NSAttributedStringKey: Any] {
         var attributes = defaultTextAttributes
         attributes[NSAttributedAutocompleteKey] = false
+        attributes[.paragraphStyle] = paragraphStyle
         return attributes
     }
+    
+    /// The NSAttributedStringKey.paragraphStyle value applied to attributed strings
+    private let paragraphStyle: NSMutableParagraphStyle = {
```
Comment (SD10) [inline MessageViewController/MessageAutocompleteController.swift]: Sorry to chime in after. Do we want to return it as a non-mutable `NSParagraphyStyle`?

Comment (BasThomas) [reply MessageViewController/MessageAutocompleteController.swift]: I guess that would make more sense, yeah

**Comment by BasThomas** [comment]
Let's close #25 and #26 for now too then and reopen if needed?

**Pull request merged**